### DISCUSSION TITLE
How can I separate .py and .pyi files into different folders?

### DISCUSSION THREAD

**Discussion by wxrayut** [microsoft/pylance-release#5214] [Q&A] [answered]
How can I to separate the .py and .pyi files into different folders, Because It looks disorganized when I have many files.
![Screenshot 2023-12-05 232716](https://github.com/microsoft/pylance-release/assets/137786429/cdac485d-acd0-4dbd-8cfd-7c567a0b62c9)

**Comment by rchiodo** [comment, marked as answer]
You can use a stubs path as described under `python.analysis.stubPath` in the [readme](https://github.com/microsoft/pylance-release#settings-and-customization). There's a default for this called `typings` that will also just work:

In your example it would be like so:

- stdlib
  - add.py
- typings
  - stdlib
     - add.pyi
